sysutils/packer/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        hashicorp packer 1.2.5 v
+
+categories          sysutils
+license             MPL-2
+maintainers         {newtonne @newtonne} openmaintainer
+platforms           darwin
+supported_archs     x86_64
+
+description         A tool for creating identical machine images for multiple \
+                    platforms from a single source configuration.
+
+long_description    ${description} \
+                    Packer is lightweight, runs on every major operating \
+                    system, and is highly performant, creating machine images \
+                    for multiple platforms in parallel. Packer comes out of \
+                    the box with support for many platforms, the full list of \
+                    which can be found at \
+                    https://www.packer.io/docs/builders/index.html.
+
+homepage            https://www.packer.io
+
+fetch.type          git
```

Comment:
I tried to get it to work using tarballs rather than git fetch, but I ran into trouble as the Makefile has a number of lines that expect to be run from a git repo. I could give it another go if you'd like?

Reply:
Oh I see. Just keep this as-is :)